### DISCUSSION TITLE
Updating npm packages

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#72] [Ideas]
I update my npm packages [this](https://www.youtube.com/watch?v=0XQXGx3lLaU) way. Do you guys follow the same practice?

If so, should we add [npm-check-updates](https://www.npmjs.com/package/npm-check-updates) to package.json?

**Comment by kentcdodds** [comment]
I do the same thing. I'd rather have this as part of the docs. Would you be willing to make a docs page about this?

**Reply by andrecasal** [reply]
Sure 👍

**Reply by andrecasal** [reply]
Made a [PR](https://github.com/epicweb-dev/epic-stack/pull/85).
